examples/espressif/iot-device-sdk-embedded-c/RELEASE-NOTES.md

```diff
@@ -0,0 +1,24 @@
+# Google Cloud IoT Core Device SDK for Embedded C version  1.0
```

Comment:
Probably better to avoid duplicating the SDK, can you use a symbolic link or other mechanism to avoid duplicate files?